docs/how-it-works/features/security/overview.md

```diff
@@ -0,0 +1,19 @@
+![binbash-logo](../../../assets/images/logos/binbash.png "Binbash"){: style="width:250px"}
+![binbash-leverage-tf](../../../assets/images/logos/binbash-leverage-terraform.png#right "Leverage"){: style="width:130px"}
+
+# AWS Reference Architecture security features
+
+??? check "VPN"
+    - [x] [VPN](./vpn.md)
+
+??? check "Services"
+    - [x] [Services](./services.md)
+
+??? check "Certificates"
+    - [x] [Certificates](./certificates.md)
+
+??? check "Audit | CloudTrail"
```

Comment:
@eze-godoy In the overview section I think we should refer to the high level sections so here we'll only present "Audit" and then add all the Audit related services as child elements. For instance if we add "Audit Manager" in the future it could be added as follows:

```
??? check "Audit"
    - [x] [CloudTrail](./audit-cloudtrail.md)
    - [x] [Audit Manager](./audit-manager.md)
```